node/pkg/logscribe/api/controller.go

```diff
@@ -0,0 +1,27 @@
+package api
+
+import (
+	"encoding/json"
+
+	"github.com/gofiber/fiber/v2"
+	"github.com/rs/zerolog/log"
+)
+
+type LogInsertModel struct {
+	Service   string          `db:"service" json:"service"`
+	Timestamp string          `db:"timestamp" json:"timestamp"`
+	Level     int             `db:"level" json:"level"`
+	Message   string          `db:"message" json:"message"`
+	Fields    json.RawMessage `db:"fields" json:"fields"`
+}
+
+func insertLogs(c *fiber.Ctx, logsChannel chan []LogInsertModel) error {
+	logEntries := new([]LogInsertModel)
+	if err := c.BodyParser(logEntries); err != nil {
+		log.Error().Err(err).Msg("Failed to parse request body")
+		return c.Status(fiber.StatusBadRequest).SendString("Failed to parse request body")
+	}
+	logsChannel <- *logEntries
```

Comment:
how about wrapping with go func

Reply:
nvm, it won't make too much difference I suppose

Reply:
yea and I think we should let the caller know if there is error in the request body

Reply:
nono I was meant for this, but it's just a single pointer so probably okay
```
go func() {logsChannel <- *logEntries}()
```